static/js/search.js

```diff
@@ -32,10 +32,10 @@ export const search = {
     panelUpdatesIndex: 0,
     // if any pannel needs a special key you need to append it here
     queryKeys: [
```

Comment:
i don't like this very much because in the query system we write things this way:
```
"name": (queryData, move) => {
            return move.name.toLowerCase().indexOf(queryData) >= 0 ? true : false
        },
        "type": (queryData, move) => {
            const types = move.types.map((x)=>gameData.typeT[x].toLowerCase())
            for (const type of types){
                if (type == queryData) return true
            }
            return false
        },
        "move-effect": (queryData, move) => {
            //it's called effect but in the data it's flags not effect
            //effect in the data might be useless to the calc, at least to short and medium terms
            const flags = move.flags.map((x)=>gameData.flagsT[x].toLowerCase())
            for (const flag of flags){
                if (flag == queryData) return true
            }
            return false
        },```
        the inconcistency here annoys me a bit, not dramatic but i sometimes forget one has to be in maj and the others does not have to be